README.md

```diff
@@ -250,6 +250,11 @@ When doing a production deployment, each step of the checklist must be performed
       to ensure that the tag is incremented properly based on the last released version.  Tags should be plain semver numbers 
       like `1.0.0` and should not have any additional prefix like `v1.0.0` or `releases/1.0.0`.  Suffixes are permitted so 
       long as they conform to the [semver spec](https://semver.org/).
+- [ ] Pushing the new tag to the git origin repository will _not_ automatically tag the Docker image.  You will need to 
+      manually tag the Docker image with the correct semver number.  Go to the [Bond project on Quay.io](https://quay.io/repository/databiosphere/bond?tab=tags)
+      and navigate to the "Tags" menu.  You should see two new images for `latest` and `develop`.  Confirm that the ages
+      of these images correspond to when you merged your changes into the `develop` branch.  On the right of one of
+      these rows, click on the gear icon and select "Add New Tag".  Enter the semver number for your release.
```

Comment:
> On the right of one of these rows, click on the gear icon and select "Add New Tag". 

I'm not sure if I'm on the right page, but I don't see the gear icon. I tried logging in with my google and github accounts. (Maybe we should specify which one to log in with?)

![Screen Shot 2021-04-28 at 12 28 45 PM](https://user-images.githubusercontent.com/5375000/116439398-7ecfd600-a81d-11eb-86d2-c7ad055b399f.png)

Reply:
That's the right page, you need to be added to the broadwrite team in quay which will grant you more permissions to the repository. Unfortunately, I don't have the power to add you to the team myself